app.js

```diff
@@ -2,20 +2,30 @@
 var express = require('express');
 var path = require('path');
 var bodyParser = require('body-parser');
+var assert = require('assert');
+var monk = require('monk');
+var server_port = process.env.OPENSHIFT_NODEJS_PORT || 8080
+var server_ip_address = process.env.OPENSHIFT_NODEJS_IP || '127.0.0.1'
+var app = module.exports = express();
+
 
-var routes = require('./routes/index');
-var port = process.env.PORT || '3000';
 
-var app = express();
+// middleware setup
 
-// view setup
-//other middleware setup
+// set up mongodb
+var uri = "mongodb://test:123@ds045464.mongolab.com:45464/courses-backend";
```

Comment:
This should probably be done as configuration either as an env variable or in a file that isn't committed. Note that there are bots that watch github for this kind of thing in order to hijack accounts.